libs/import/src/lib/view-extracted-elements/titles/view-extracted-titles.component.scss

```
@@ -0,0 +1,3 @@
+section:nth-child(2) {
+  margin-top: 24px;
+}
```

It's better to have something like that
```suggestion
:host {
  display: flex;
  flex-direction: column;
  gap: 24px;
}
```